assets/scss/templates/_guidelines.scss

```diff
@@ -133,17 +133,27 @@
   }
 }
 /* buttons */
-
 .main-buttons {
   display: flex;
-  flex-wrap: wrap;
   padding-top: 20px;
+  justify-content: space-between;
+
+  .buttons-column {
+    display: flex;
+    flex: 1;
+    flex-wrap: wrap;
+    align-items: flex-start;
+
+    &:nth-child(2) {
+      height: 70%;
+    }
+  }
 }
 
 .buttons-card {
   border-radius: 5px;
   margin-bottom: 30px;
-  width: 25%;
+  width: 290px;
```

Comment:
maybe this should be in em, so it will depend on the container of these buttons-cards